plutus-core/untyped-plutus-core/test/Evaluation/Golden/caseNonTag.plc.golden

```diff
@@ -0,0 +1,3 @@
+(Left An error has occurred:  error:
+A non-constructor value was scrutinitzed in a case expression
+Caused by: (con integer 1))
```

Comment:
Could there be any possible benefit from allowing `case` to scrutinize built-in enumeration types such as `Bool`, `List`, `Data` etc?

Reply:
I can't think of any, I hate it :p